caffe2/operators/batch_gather_ops.cu

```diff
@@ -16,8 +16,16 @@ bool BatchGatherOp<CUDAContext>::RunOnDevice() {
 template <>
 template <typename TInd>
 bool BatchGatherOp<CUDAContext>::DoRunWithType() {
+    return DispatchHelper<TensorTypes2<int8_t,int16_t,int32_t,int64_t,
+           uint8_t,at::Half,long,float,double,GenericTensorImplementation>,
```

Comment:
this has both `long` and `int64_t`?

Reply:
I added long to solve the original problem with the alexnet example, the original implementation allowed the types int32 and int64 as valid inputs, I think I just copy-pasted them - should I remove int64 and leave long?